<launch>
    <node name="fly_node" pkg="tracks" type="fly_node.py" output="screen">
        <param name="lon" type="double" value="68.993910"/>
        <param name="lat" type="double" value="61.001389"/>
        <param name="alt" type="double" value="60"/>
        <param name="satname" type="string" value="NOAA 19"/>
        <param name="debug" type="bool" value="false"/>
    </node>
</launch>
